docs/resource/host.md

```diff
@@ -31,6 +31,19 @@
 | `x86` | 32-bit x86 |
 <!-- endsemconv -->
 
+**type:** `host.cpu`
+
+<!-- semconv host.cpu -->
+| Attribute  | Type | Description  | Examples  | Requirement Level |
+|---|---|---|---|---|
+| `host.cpu.vendor.id` | string | Processor manufacturer identifier. A maximum 12-character string. | `GenuineIntel` | Opt-In |
+| `host.cpu.family` | int | Numeric value specifying the family or generation of the CPU. | `6` | Opt-In |
+| `host.cpu.model.id` | int | Model identifier. It provides more granular information about the CPU, distinguishing it from other CPUs within the same family. | `6` | Opt-In |
+| `host.cpu.model.name` | string | Model designation of the processor. | `11th Gen Intel(R) Core(TM) i7-1185G7 @ 3.00GHz` | Opt-In |
+| `host.cpu.stepping` | int | Stepping or core revisions. | `1` | Opt-In |
+| `host.cpu.cache.l2.size` | int | The amount of level 2 memory cache available to the processor (in Bytes). | `12288000` | Opt-In |
```

Comment:
Is the `Opt-In` a deliberate choice these new attributes? Do you think it aligns with the description of Opt-in we have? 
https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/common/attribute-requirement-level.md#opt-in

Reply:
I would consider those as more "advanced" attributes that might not be of importance for all use-cases, that's why I went for the safe Opt-In for now.
I would be ok to change it though.
@mx-psi @frzifus any thoughts on this?

Reply:
> I would consider those as more "advanced" attributes that might not be of importance for all use-cases, that's why I went for the safe Opt-In for now.

I think so to. The reason I'm asking is because for Opt-In, the "instrumentation" would need to offer a way for users to opt-in into these attributes explicitly. Is something like this possible in the collector today?

Reply:
Perfect!